tests/my_app/templates/base.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title><!-- TODO: A great title --></title>
+    </head>
+    <body>
+        {% block content %}{% endblock %}
+    </body>
+</html>
```

Comment:
Don't think this is needed. 🤔

Reply:
No, I don't think so, I am equally tired/lazy and copied the existing tests app you had :D. Let me remove that now.